audit-log-api/src/use-cases/RetryMessageUseCase.ts

```diff
@@ -1,21 +1,23 @@
 import type { PromiseResult } from "shared"
-import { isError } from "shared"
-import type GetLastEventUseCase from "./GetLastEventUseCase"
+import { decodeBase64, isError } from "shared"
+
+import type GetLastFailedMessageEventUseCase from "./GetLastEventUseCase"
 import type RetrieveEventXmlFromS3 from "./RetrieveEventXmlFromS3UseCase"
 import type CreateRetryingEventUseCase from "./CreateRetryingEventUseCase"
 import type SendMessageToQueueUseCase from "./SendMessageToQueueUseCase"
 import validateEventForRetry from "./validateEventForRetry"
 
 export default class RetryMessageUseCase {
   constructor(
-    private readonly getLastEventUseCase: GetLastEventUseCase,
+    private readonly getLastFailedMessageEventUseCase: GetLastFailedMessageEventUseCase,
     private readonly sendMessageToQueueUseCase: SendMessageToQueueUseCase,
     private readonly retrieveEventXmlFromS3UseCase: RetrieveEventXmlFromS3,
     private readonly createRetryingEventUseCase: CreateRetryingEventUseCase
   ) {}
 
   async retry(messageId: string): PromiseResult<void> {
-    const event = await this.getLastEventUseCase.get(messageId)
+    console.log(`Retrying message ${messageId}`)
```

Comment:
Is there a ticket to add a proper logger to this repo 🤔

Reply:
If there isn't, there should be. I agree console logging like this isn't ideal. I don't even know if we parse the lambda logs into ES/Kibana at all at the moment? Without the statements I've added here, the logs are empty.